src/policy/space.rs

```diff
@@ -524,6 +528,7 @@ pub struct CommonSpace<VM: VMBinding> {
     /// This field equals to needs_log_bit in the plan constraints.
     // TODO: This should be a constant for performance.
     pub needs_log_bit: bool,
+    pub needs_satb: bool,
```

Comment:
I added this because `needs_log_bit` has a generational semantic associated with it.

Reply:
`needs_satb` is now removed. https://github.com/mmtk/mmtk-core/pull/1364 should have fixed issues with log bits.